feature-libs/organization/account-summary/components/account-summary/details/header/header.component.ts

```diff
@@ -0,0 +1,119 @@
+import { Component, OnInit } from '@angular/core';
+import { Address, TranslationService } from '@spartacus/core';
+import {
+  AccountSummaryDetails,
+  AccountSummaryFacade,
+} from '@spartacus/organization/account-summary/root';
+import { Card } from '@spartacus/storefront';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+const notApplicable = 'n/a';
+
+@Component({
+  selector: 'cx-account-summary-header',
+  templateUrl: './header.component.html',
+})
+export class HeaderComponent implements OnInit {
+  headerDetails$: Observable<AccountSummaryDetails>;
+
+  constructor(
+    private accountSummaryFacade: AccountSummaryFacade,
+    protected translation: TranslationService
+  ) {}
+
+  ngOnInit(): void {
+    this.headerDetails$ = this.accountSummaryFacade.getAccountSummary();
+  }
+
+  getIdCardContent(id?: string): Observable<Card> {
+    return this.translation.translate('orgAccountSummary.details.uid').pipe(
+      map((idTitle) => ({
+        title: idTitle,
+        text: [id || notApplicable],
+      }))
+    );
+  }
+
+  getNameCardContent(name?: string): Observable<Card> {
+    return this.translation.translate('orgAccountSummary.details.name').pipe(
+      map((nameTitle) => ({
+        title: nameTitle,
+        text: [name || notApplicable],
+      }))
+    );
+  }
+
+  getAddressCardContent(billingAddress?: Address | any): Observable<Card> {
+    return this.translation.translate('orgAccountSummary.details.address').pipe(
+      map((addressTitle) => {
+        const name =
+          billingAddress?.fullnameWithTitle ?? // sometimes fullnameWithTitle is not set
```

Comment:
is `fullnameWithTitle` in `Address's DTO`? If yes, then add it directly into `Address`, or augment `Address` model.

Reply:
fullnameWithTitle no longer part of Address Model